tests/test_behavior/test_movie_interface.py

```diff
@@ -248,11 +249,10 @@ def test_movie_regular_timestamps(self):
             Movie=dict(starting_times=self.starting_times, timestamps=timestamps, external_mode=True)
         )
 
-        rate = 1.0 / np.round(timestamps[1] - timestamps[0], 9)
         fps = 25.0
         expected_warn_msg = (
-            f"The fps={fps} from movie data is unequal to the difference in "
-            f"regular timestamps. Using fps={rate} from timestamps instead."
+            "The fps=25.0 from movie data is unequal to the difference in "
+            "regular timestamps. Using fps=5.0 from timestamps instead."
```

Comment:
@CodyCBakerPhD 
Looking at the CI run, this should either be fps=5.000000000000007, or should use rate = 1.0 / (timestamps[1] - timestamps[0]), but I can see why you removed that before, but hopefully how rate is calculated won't be changed soon.

Reply:
Right, that's just a display issue though(with the print statement). Fixing now